server/lib/src/be/mod.rs

```diff
@@ -2121,7 +2126,7 @@ impl Backend {
         debug!(db_tickets = ?cfg.pool_size, profile = %env!("KANIDM_PROFILE_NAME"), cpu_flags = %env!("KANIDM_CPU_FLAGS"));
 
         // If in memory, reduce pool to 1
-        if cfg.path.is_empty() {
+        if cfg.path == Path::new("") {
```

Comment:
again. `.as_os_str().is_empty()` seems to be the more common approach [ref](https://github.com/rust-lang/rust/issues/30259#issuecomment-191033337)